assets/products/product_clients/macbookair.py

```diff
@@ -0,0 +1,36 @@
+from assets.products.base_product import BaseProduct
+from assets.sources.source_clients.amazon import Amazon
+from assets.sources.source_clients.best_buy import BestBuy
+from assets.sources.source_clients.newegg import NewEgg
+from assets.sources.source_clients.walmart import Walmart
+from assets.sources.source_clients.b_and_h import BandH
+
+class MacBookAir2020(BaseProduct):
+    best_buy = "https://www.bestbuy.com/site/apple-macbook-air-13-3-laptop-with-touch-id-intel-core-i3-8gb-memory-256gb-solid-state-drive-latest-model-gold/6366612.p?skuId=6366612"
+    walmart = "https://www.walmart.com/ip/Apple-13-3-MacBook-Air-2020-Gold-Wallet-Reader-MagicMouse2-USBHUB/201580499"
+    amazon = "https://www.amazon.com/Apple-MacBook-13-inch-256GB-Storage/dp/B08632W2H6?th=1"
```

Comment:
I tried running this with a similar test to the one you posted for the hp chromebook. When the scraper is run for amazon I am getting the following error for line 23 in amazon.py:
`AttributeError: 'NoneType' object has no attribute 'text'`
I am guessing that amazon blocked the request.

Reply:
That's a decent guess. I will look into this then re-assign once I have a solution. Weird that it worked for me then didn't for you. Possible that it went out of stock?